sympy/printing/preview.py

```diff
@@ -183,8 +183,12 @@ def preview(expr, output='png', viewer=None, euler=True, packages=(),
     try:
         workdir = tempfile.mkdtemp()
 
-        with open(join(workdir, 'texput.tex'), 'w', encoding='utf-8') as fh:
-            fh.write(latex_main % latex_string)
+        try:
+            with open(join(workdir, 'texput.tex'), 'w', encoding='utf-8') as fh:
+                fh.write(latex_main % latex_string)
+        except TypeError:  # Python 2's open() does not support encoding
```

Comment:
@bjodah, I tested the new patch and I don't think it works as you expect. It will definitely use encoding support on Python 3+ where open defaults to io.open. However, in Python 2.7, one must import io before it overrides the default open method. That said, Python 2.6 doesn't support io.open at all. The io module only being released in 2.7 or greater. 

I've replaced the whole open code as follows and this appears to work and should properly fallback to codec.open in Python 2.6 (this part untested).

```python
        try:
            from io import open
            with open(join(workdir, 'texput.tex'), 'w', encoding='utf-8') as fh:
                fh.write(latex_main % latex_string)
        except:
            # use 2.6 codecs
            from codecs import open
            with open(join(workdir, 'texput.tex'), mode='w', encoding='utf-8') as fh:
                fh.write(latex_main % latex_string)
```